api/napi/src/interpreter/component_instance.rs

```diff
@@ -168,6 +272,63 @@ impl JsComponentInstance {
             .map_err(|_| napi::Error::from_reason("Cannot invoke callback."))?;
         super::to_js_unknown(&env, &result)
     }
+
+    #[napi]
+    pub fn invoke_global(
+        &self,
+        env: Env,
+        global_name: String,
+        callback_name: String,
+        arguments: Vec<JsUnknown>,
+    ) -> Result<JsUnknown> {
+        let ty = self
+            .inner
+            .definition()
+            .global_properties_and_callbacks(global_name.as_str())
+            .ok_or(napi::Error::from_reason(format!("Global {global_name} not found")))?
+            .find_map(|(name, proptype)| if name == callback_name { Some(proptype) } else { None })
+            .ok_or(())
+            .map_err(|_| {
+                napi::Error::from_reason(
+                    format!(
+                        "Callback {} of global {global_name} not found in the component",
```

Comment:
Please add a test that covers this code path (you can fold that into the existing test).